src/mailchimp/mailchimp_manager.py

```diff
@@ -8,13 +8,18 @@
     * Implement methods
     * Implement datalogging
 """
+import mailchimp_marketing as MailchimpMarketing
+
+from mailchimp_marketing.api_client import ApiClientError
```

Comment:
If you're including new libraries, you need to add them to the `requirements.txt` file. Make sure to specify the version number.